src/views/Dashboard/Customer/Home/PaymentHistory/index.vue

```diff
@@ -86,10 +86,12 @@ export default {
 
   async created() {
     await this.metamaskDispatchAction(this.onSearchInput)
+    console.log(this.payments, "<== payment")
```

Comment:
this line should be remove from the code

Reply:
deleted